config/manager/kustomization.yaml

```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: gresearch/armada-operator
-  newTag: latest
+  newName: armadaproject/armada-operator
```

Comment:
Wasn't "latest" cleaner here?

Reply:
2 options:
* use latest
* update to hardcoded latest semver on each release

Reply:
Needing to update this seems likely to cause future issues to me....